week1/index.js

```diff
@@ -1 +1,34 @@
-console.log("Hello world")
\ No newline at end of file
+import HTTP from 'http'
+import Path from 'path'
+import stateHTML from './responses/stateHTML'
+import addHTML from './responses/addHTML'
+import removeHTML from './responses/removeHTML'
+import resetHTML from './responses/resetHTML'
+import notFound from './responses/notFound'
+
+
+var state = 10;
+
+const server = HTTP.createServer((request, response) => {
+    console.log(request.method, request.url)
+    switch (request.url) {
+        case '/state':
+            stateHTML(response, state)
+            break
+        case '/add':
+            addHTML(response, state)
+            break
+        case '/remove':
+            removeHTML(response, state)
+            break
+        case '/reset':
+            resetHTML(response, state)
+            break
+        default:
+            response.statusCode = 404
+            notFound(response, "File not found")
+            break
+    }
+    response.end()
+})
+server.listen(8080)
```

Comment:
You could set your editor to automatically add [new line](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) at the end of the file on save!